### DISCUSSION TITLE
impossible to read customers blank xml in webservice

### DISCUSSION THREAD

**Discussion by LilianDubois** [PrestaShop/PrestaShop#32434] [Q&A]
Hello everyone, I have a problem with the webservices when I try to access the link: http://localhost:8888/prestashop/api/customers?schema=blank
I am told that there is an error: 

![image](https://user-images.githubusercontent.com/82089120/236072681-bc617e41-e330-4e30-a96a-a56a468ebc81.png)

while all the other links to access the other xml are working fine 

Could someone help me?

**Comment by kpodemski** [comment]
You should see the exact output you have returned, maybe there's some error in the code.

**Comment by DJ2412** [comment]
@LilianDubois Did you manage to solve it?
I have the same problem.
<img width="292" alt="image" src="https://github.com/PrestaShop/PrestaShop/assets/39198090/fe9ed680-6595-44ea-934c-673fd69ddb9c">
